src/posts/2023-07-04-cilium-cni/index.md

```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-07-04T11:28:00.000Z'
+title: 'Cilium CNI: A Comprehensive Deep Dive Guide for Networking and Security Enthusiasts!'
+ogImage: ogimage.png
+ogSummary: 'Dive into this step-by-step guide onCilium, and how it works'
```

Comment:
This is the same description as above

Reply:
thanks for catching that, must have missed that.